ActionSheet Popover: update demo description (#2152)
DevExpress/devextreme-demos: JSDemos/Demos/ActionSheet/PopoverMode/description.md

> ActionSheet can appear from the bottom of the page or over a certain element. Enable the [usePopover](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#usePopover) property to display ActionSheet over a certain element on the page and use the [target](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#target) property to specify this element. In this demo, ActionSheet is integrated with the List component and appears when you click a list item.

Reviewer: ```suggestion
ActionSheet can appear at the bottom of the page or over a certain element. Enable the [usePopover](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#usePopover) property to display ActionSheet over a certain element on the page and use the [target](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#target) property to specify this element. In this demo, ActionSheet is integrated with the List component and appears when you click a list item.
```

> The ActionSheet component is a pop-up sheet that contains a set of buttons. The buttons allow users to perform custom actions related to a single task (call, send a message, delete, or edit a selected contact). This demo shows how to create a simple ActionSheet in the **popover** mode and handle button clicks.
> 
> ## Specify Display Mode
> 

Reviewer 2: ```suggestion
```

> ActionSheet can appear from the bottom of the page or over a certain element. Enable the [usePopover](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#usePopover) property to display the action sheet over a certain element on the page, and use the [target](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#target) property to specify this element.

Reviewer 2: ```suggestion
ActionSheet can appear from the bottom of the page or over a certain element. Enable the [usePopover](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#usePopover) property to display ActionSheet over a certain element on the page and use the [target](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#target) property to specify this element. In this demo, ActionSheet is integrated with the List component and appears when you click a list item.
```
